model/cluster_message.go

```diff
@@ -26,6 +26,11 @@ const (
 	CLUSTER_EVENT_INVALIDATE_CACHE_FOR_SCHEMES                      = "inv_schemes"
 	CLUSTER_EVENT_CLEAR_SESSION_CACHE_FOR_ALL_USERS                 = "inv_all_user_sessions"
 
+	// Plugin Events
+	CLUSTER_EVENT_INSTALL_PLUGIN = "install_plugin"
+	CLUSTER_EVENT_REMOVE_PLUGIN  = "remove_plugin"
```

Comment:
It's a minor nit, but the lack of symmetry between `install` and `remove` irritates me. Would it make sense to converge on `install` and `uninstall` everywhere?

Reply:
I agree. I personally prefer `uninstall` as well. Let's fix it up :).